addons/web/static/tests/helpers/mock_server.js

```diff
@@ -771,14 +770,13 @@ export class MockServer {
             }
         }
         function formatValue(groupByField, val) {
-            let [fieldName, aggregateFunction] = groupByField.split(":");
-            aggregateFunction = aggregateFunction || "month";
-            if (fields[fieldName].type === "date") {
-                // what about datetime?
-                if (!val) {
-                    return false;
-                }
-                const date = DateTime.fromISO(val);
+            if (!val) {
+                return false;
```

Comment:
does not work for type "integer" for example

Reply:
true !